include/inviwo/core/datastructures/camera/camera.h

```diff
@@ -200,6 +203,9 @@ class IVW_CORE_API Camera : public Serializable {
 inline const vec3& Camera::getLookFrom() const { return lookFrom_; }
 inline const vec3& Camera::getLookTo() const { return lookTo_; }
 inline const vec3& Camera::getLookUp() const { return lookUp_; }
+inline vec3 Camera::getLookRight() const {
+    return glm::cross(glm::normalize(getDirection()), lookUp_);
```

Comment:
Not guaranteed that `lookUp_` is normalized.
```suggestion
    return glm::cross(glm::normalize(getDirection()), glm::normalize(lookUp_));
```